unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/PlacesAndEventsSectionComponentView.cs

```diff
@@ -109,11 +124,17 @@ internal void CreateSubSectionSelectorMappings()
             eventsSubSection.SetActive(isActive);
             currentSelectedIndex = EVENTS_SUB_SECTION_INDEX;
         });
+        subSectionSelector.GetSection(FAVORITES_SUB_SECTION_INDEX)?.onSelect.AddListener((isActive) =>
```

Comment:
They can be null because there might be an error in the configuration of the component that holds them, to avoid a full failure of the script there is the constant null check